src/config.py

```diff
@@ -320,11 +320,22 @@ def _build_destination(
                 f' "{dest_config["ref"]}" defined in config'
             ) from e
 
+        try:
+            request_timeout = dest_config["timeout"]
+            if request_timeout < 1:
+                raise ValueError("Request timeout for DuneClient cannot be < 1")
+            elif type(request_timeout) != int:
+                raise ValueError("Request timeout must be int")
```

Comment:
The order should probably be reversed (check the type with priority, then check the value). Otherwise the error message could be weird (or not caught at all) if the user specified for instance a string.

Reply:
Also I think the config parser might interpret everything as string initially so we might want to just see if the int cast passes here or not.

Reply:
I set it up now to try casting the request_timeout value to an integer, and throw an error if that fails.